docs/connectors/rhoas-cli-getting-started-connectors/README.adoc

```diff
@@ -118,38 +119,38 @@ $ rhoas kafka create --name my-kafka-instance
 +
 [source,subs="+quotes"]
 ----
-$ rhoas kafka topic create --name test-topic 
+$ rhoas service-account create --file-format=json --short-description=test-service-account 
 ----
 
-** Create a service account and copy the service account ID and secret. You can use the service account to connect and authenticate your {product-connectors} instances with your Kafka instance.
+** For your Kafka instance, set the permissions for the service account to enable {connectors} instances (that are configured with the service account credentials) to produce and consume messages in any topic in the Kafka instance.
 +
 [source,subs="+quotes"]
 ----
-$ rhoas service-account create --file-format json --short-description="test-service-account" 
+$ rhoas kafka acl grant-access --producer --consumer --service-account=<service-acct-id> --topic all --group all 
 ----
 
-** For your Kafka instance, set the *Consume from a topic* and *Produce to a topic* permissions for the service account to `ls \*`. The `is \*` settings enable {product-connectors} instances that are configured with the service account credentials to produce and consume messages in any topic in the Kafka instance.
+** Create a Kafka topic named `test-topic`. The Kafka topic stores messages sent by producers (data sources) and makes them available to consumers (data sinks).
 +
 [source,subs="+quotes"]
 ----
-$ rhoas kafka acl grant-access --producer --consumer --service-account <service-acct-id> --topic all --group all 
+$ rhoas kafka topic create --name=test-topic 
 ----
 
 [id="proc-create-connector-namespace_{context}"]
-== Creating a namespace to host your {product-connectors} instances
+== Creating a namespace to host your {connectors} instances
 [role="_abstract"]
 
-A Connectors namespace hosts your Connectors instances. 
+A {connectors} namespace hosts your {connectors} instances. 
 
-The namespace that you use depends on your OpenShift Dedicated environment.
+The namespace that you use depends on your OpenShift Dedicated (OSD) environment.
 
-* If you're using a trial cluster in your own OpenShift Dedicated environment:: The namespace is created when you add the {product-connectors} service to your trial cluster, as described in https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/15a79de0-8827-4bf1-b445-8e3b3eef7b01[Adding and removing the Red Hat OpenShift Connectors add-on on your OpenShift Dedicated trial cluster^]. Your OSD trial cluster namespace is active for 60 days.
+If you're using a trial cluster in your own OSD environment:: The namespace is created when you add the {product-connectors} service to your trial cluster, as described in https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/15a79de0-8827-4bf1-b445-8e3b3eef7b01[Adding the Red Hat OpenShift {connectors} add-on to your OpenShift Dedicated trial cluster^]. Your OSD trial cluster namespace is active for 60 days.
```

Comment:
OSD might be not well known acronim

Reply:
True - the first instance (see line 149) defines it OpenShift Dedicated (OSD)

Reply:
Actually, I need to retract one of my earlier comments. I think OSD is not an approved abbreviation, so we should not use it.